sound/soc/sof/ipc4-debug-slot-debugfs.c

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2024 Intel Corporation.
```

Comment:
2025?

Reply:
The file was created, the commit made, and the PR pushed to public github on 2024. Should the PRs always be edited to reflect the probable merge date? But I'll update it to 2024-2025 once we agree one the file and function names.